packages/text-editor/src/components/extension/nodeHighlight.ts

```diff
@@ -18,6 +18,7 @@ export function highlightUpdateCommand (): TextEditorCommand {
 
 export interface NodeHighlightExtensionOptions extends NodeUuidOptions {
   getNodeHighlightType: (uuid: string) => NodeHighlightType | undefined | null
+  isActiveNode?: (uuid: string) => boolean
```

Comment:
Why do we need another callback for this? As long as all highlight is based on callback(s), can we return all required information from `getNodeHighlightType`?

Reply:
We would have to create separate types like : `WARNING` & `WARNING_ACTIVE`, `ADD` && `ADD_ACTIVE` etc.
I am not sure how it would be better.